public/scripts/view-util.js

```diff
@@ -355,14 +355,22 @@ var ViewUtil = (function($, DataUtil) {
     }
 
     function renderIo(step) {
-	return '<div style="font-size:10px">' +
+	var io_html = '<div style="font-size:10px">' +
 	    '<table class="table table-centered table-striped">' +
 	    '<tr> <th>I/O Type</th> <th>Read</th> <th>Written</th> </tr>' +
 	    '<tr> <td>HDFS</td> ' +
 	    '<td>' + view.prettyPrintBytes(step.hdfs_bytes_read) + '</td>' +
 	    '<td>' + view.prettyPrintBytes(step.hdfs_bytes_written) + '</td>' +
-	    '</tr>' +
-	    '<tr> <td>Tuples</td> ' +
+	    '</tr>';
+
+	if (step.gs_bytes_read !== null || step.gs_bytes_written !== null) {
```

Comment:
Since this section has some of the most valuable information on the page, I wanted to avoid unnecessary filler for non-GCS users.